docs/source/index.rst

```diff
@@ -41,13 +41,16 @@ DataFusion offers SQL and Dataframe APIs, excellent
 CSV, Parquet, JSON, and Avro, extensive customization, and a great
 community.
 
-To get started with examples, see the `example usage`_ section of the user guide and the `datafusion-examples`_ directory.
+To get started, see
```

Comment:
I also added a link to the library user guide as well as making this into a bullet list so it is easier to see what options are available